Casks/font/font-m/font-migmix-1m.rb

```diff
@@ -9,10 +9,7 @@
 
   livecheck do
     url :homepage
-    strategy :page_match do |page|
-      page.scan(/href=.*migmix-1m[._-]v?(\d+(?:\.\d+)*)\.zip"/i)
-          .map { |match| match[0].insert(4, ".") }
-    end
+    regex(%r{href=.*?/download/v?(\d+(?:\.\d+)+)/migmix-1m}i)
```

Comment:
```suggestion
    regex(%r{href=.*?/download/v?(\d+(?:\.\d+)+)/migmix-1m[._-]}i)
```

Since we're matching URLs like https://github.com/itouhiro/mixfont-mplus-ipa/releases/download/v2020.0307/migmix-1m-20200307.zip, there may be something to be said for matching the delimiter at the end of these filename prefixes. This will ensure that we only match what we intend if upstream creates additional variants that start with a similar prefix but have something else afterward.

For example, if upstream were to create a new variant named something like `migmix-1m2`, `migmix-1m` in the regex would also erroneously match versions for that variant. We only catch this type of issue if/when it fails on CI or someone notices an unexpected version and manually verifies that it's not appropriate, so this little tweak could save us some effort down the road. It may never become an issue but better safe than sorry.

Reply:
Thanks!